dotnet/src/Plugins/Plugins.Web/Bing/BingMetaTag.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Text.Json.Serialization;
+
+namespace Microsoft.SemanticKernel.Plugins.Web.Bing;
+
+/// <summary>
+/// Defines a webpage's metadata.
+/// </summary>
+public sealed class BingMetaTag
```

Comment:
We can add an internal default constructor `internal BingMetaTag()` to limit model creation to SK functionality only. This way, only SK can create the class, allowing more flexibility for future changes. The same applies to other classes that SK consumer code accesses but is not supposed to create.